src/mono/mono/mini/interp/interp.c

```diff
@@ -4205,39 +4173,41 @@ mono_interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClause
 				LOCAL_VAR (call_args_offset, gpointer) = unboxed;
 			}
 
-			InterpMethodCodeType code_type = cmethod->code_type;
+jit_call:		
```

Comment:
I would keep an eye open for potential regressions on wasm due to more complicated control flow which might hinder some compiler optimizations.